src/main/java/com/yanolja_final/domain/order/controller/OrderController.java

```diff
@@ -20,12 +25,22 @@ public class OrderController {
 
     private final OrderFacade orderFacade;
 
+    @GetMapping
+    public ResponseEntity<ResponseDTO<Map<String, Object>>> read(
+        @LoginedUserId Long userId,
+        @PageableDefault(size = 10) Pageable pageable
```

Comment:
```suggestion
        Pageable pageable
```
기본값이 이미 10이기 때문에 어노테이션 제거하셔도 됩니다

Reply:
참고: https://github.com/yanolja-finalproject/Backend/blob/develop/src/main/resources/application.yml
spring.data.web.pageable.default-page-size